cmd/execution_builder.go

```diff
@@ -572,11 +558,7 @@ func (exeNode *ExecutionNode) LoadExecutionState(
 ) {
 
 	chunkDataPacks := storage.NewChunkDataPacks(node.Metrics.Cache, node.DB, node.Storage.Collections, exeNode.exeConf.chunkDataPackCacheSize)
-
-	// Needed for gRPC server, make sure to assign to main scoped vars
-	exeNode.events = storage.NewEvents(node.Metrics.Cache, node.DB)
-	exeNode.serviceEvents = storage.NewServiceEvents(node.Metrics.Cache, node.DB)
-	exeNode.txResults = storage.NewTransactionResults(node.Metrics.Cache, node.DB, exeNode.exeConf.transactionResultsCacheSize)
+	node.Storage.TransactionResults = storage.NewTransactionResults(node.Metrics.Cache, node.DB, exeNode.exeConf.transactionResultsCacheSize)
```

Comment:
this seems sketchy.  someone could accidentally hold a reference to the original transaction result, before it is replaced by this.  Instead of replacing the transaction result, perhaps we should apply the config on the original transaction result?